src/platforms/capacitor/index.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Capacitor
+---
+
+On this page, we get you up and running with Sentry's Capacitor SDK, so that it will automatically report errors and exceptions in your application.
+
+The Capacitor SDK can be used with any framework that supports Capacitor.
+
+## Install
+
+The Capacitor SDK is currently available for Android devices.
+To install the Capacitor SDK, use npm or yarn:
+
+```bash {tabTitle:npm}
+npm install --save @sentry/capacitor @sentry/tracing
+```
+
+```bash {tabTitle:Yarn}
+yarn add @sentry/capacitor @sentry/tracing
+```
```

Comment:
the idea is not to duplicate the content, but to `include` from the include files eg:

`<PlatformContent includePath="getting-started-install" />`

this will tell the docs to pick up the `getting-started-install.mdx` and render it here, the same for the other blocks of this file